config.example.toml

```diff
@@ -1,50 +1,42 @@
 # Controls deployment options
 [deployment]
-    host = "127.0.0.1"
+    host = "0.0.0.0" # Set to 127.0.0.1 if you use a reverse proxy, or access it locally
     port = 8000
 
     # Amount of worker Priviblur instances to spawn. Increases speed significantly.
     # workers = 1
 
-    # # If you're running Priviblur behind a remote proxy, one or more of the following must be set
-    # # can also be set via env variables by captialzing and prefixing with PRIVIBLUR_
-    # #
-    # # For more information see
-    # # https://sanic.dev/en/guide/advanced/proxy-headers.html
-    # #
-    # # Default: None
-    # #
-    # forwarded_secret =
-    # real_ip_header =
-    # proxies_count =
-
-# Controls redis cache options
-# Ignore to disable the cache
-#
-# [cache]
+
+# # Controls redis cache options
+# # Ignore to disable the cache
+# # 
+# [cache] # To enable caching Redis is required, uncomment every lines to it
```

Comment:
By the way @syeopite I'm not sure "uncomment every lines to it" (I'll fix that typo by that way)  is true, I assumed it is, but maybe if the URL is set the others will just use their default value (written here), am I wrong?

Reply:
Only the url is required to enable caching, the others will just use their default values when unset.

Reply:
Alright, I'll clarify that

Reply:
Done